Sources/SwiftDocC/Semantics/ReferenceResolver.swift

```diff
@@ -344,6 +344,12 @@ struct ReferenceResolver: SemanticVisitor {
         let newDeprecationSummary = article.deprecationSummary.flatMap {
             visitMarkupContainer($0) as? MarkupContainer
         }
+        // If there's a call to action with a local-file reference, change its context to `download`
+        if let downloadFile = article.metadata?.callToAction?.file,
+            var resolvedDownload = context.resolveAsset(named: downloadFile.path, in: bundle.rootReference) {
+            resolvedDownload.context = .download
```

Comment:
Is this effectively overriding the case where the user provides a local file but with a `link` purpose? (That seems like the right behavior for me – just checking.)

Reply:
Basically yeah; this makes it so that if someone uses the `file` argument to `@CallToAction`, that file is treated as a download instead of whatever it might otherwise have been. (This let me test using an SVG file as a download, which otherwise is treated as an image asset.)

Reply:
Makes sense. Thank you!